tests/Framework.py

```diff
@@ -356,7 +356,7 @@ def ignoreWarning(self, category=Warning, module=""):
             yield
 
     def __openFile(self, mode):
-        for (_, _, functionName, _) in traceback.extract_stack():
```

Comment:
thisis caused by newer version of black, run pre-commit would be a better solution.